Dockerfile

```diff
@@ -13,11 +13,9 @@ RUN yum -y install epel-release \
     && yum -y install ansible sudo \
     && ansible-galaxy install -p /opt/setup/roles -r requirements.yml
 
-ARG OMERO_VERSION=5.6.0-m1
-ARG OMEGO_ADDITIONAL_ARGS=
```

Comment:
Why is this removed?

Reply:
We're not using omego

Reply:
Can you outline how to do this then?

Reply:
If there's a URL to a zip somewhere I can update the role. Would it just be for OMERO.web, or for OMERO.py, OMERO.marshal, etc?

Reply:
The previous functionality encompassed py, web, dropbox, marshal.

Reply:
I'd add this as a use case to consider in https://github.com/ome/ansible-role-omero-web/issues/27

Reply:
:+1: see also https://github.com/ome/omero-web-docker/pull/33

Reply:
--> https://github.com/ome/ansible-role-omero-web/issues/29